### ISSUE TITLE
Conversation history

### ISSUE THREAD

**Issue by nelsonic** [FAC-GM/gm-magic#13]
@SimonLab can you see a conversation history in the :star: **Relationship** tab?
![simon-message-history](https://cloud.githubusercontent.com/assets/194400/10856210/0a563aa2-7f3e-11e5-8152-6cf8f8a7d13c.png)

**Comment by SimonLab** [comment]
@nelsonic yes I have the message now. However you can only see the responses, not the messages you sent, it makes the conversation difficult to follow but maybe it's not the point of the relationship tab.
![relationship](https://cloud.githubusercontent.com/assets/6057298/10868467/4602eb3c-8087-11e5-8861-a252eb8242a6.png)

**Comment by nelsonic** [comment]
Agreed, only having one side of the conversation history is Lame.
I think the "full" conversation history is only available if you *pay*...
We can still get the fact that a conversation exists which is *way* more useful than various people contacting the same person multiple times ...

**Comment by iteles** [comment]
Agreed! I think the details of the conversation would be interesting, but the essential thing is for the organisation to present a united front and at least give the impression that _everyone knows what everyone else is doing_ rather than 3 people introducing themselves to the same person on linkedIn :wink:
